src/github/client/github-client.ts

```diff
@@ -99,10 +99,24 @@ export default class GitHubClient {
 		return response;
 	}
 
+	private async graphql<T>(query: string): Promise<AxiosResponse> {
+		return await this.axios.post<T>("https://api.github.com/graphql",
+			{
+				query
+			},
+			{
+				...await this.installationAuthenticationHeaders(),
+				params: {
+					installationId: this.githubInstallationId,
+				}
+			});
+		//TODO: error handling
```

Comment:
@thombergs Shouldn't error handling be the responsibility of the consumer code? 

Axios client throws exceptions if which we wrap into GithubClientError, and it is applied here as well. Let's remove this TODO.

Reply:
yes, this is a copy&paste artefact from the `get()` function